lib/createsend/journey_email.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+
+import json
+
+from createsend.createsend import CreateSendBase
+from createsend.utils import json_to_py
+
+
+class JourneyEmail(CreateSendBase):
+    """Represents a journey and associated functionality"""
+
+    def __init__(self, auth=None, journey_email_id=None):
+        self.journey_email_id = journey_email_id
+        super(JourneyEmail, self).__init__(auth)
+
+    def bounces(self, date="", page=1, page_size=1000, order_field="date", order_direction="asc"):

```

Comment:
Why is the default page size 1000? Does the API have a default that will take effect if we don't set it? Because we should go with that (then the default can be changed on the server side).

Reply:
The API default is 1000 (seemingly arbitrarily)